### DISCUSSION TITLE
[Question][Hw3] make command not work

### DISCUSSION THREAD

**Discussion by hsKim25** [prosyslab-classroom/cs348-information-security#514] [Q&A]
When I tried make command in the root repository,

There is error with error message:

File "src/dune", line 11, characters 38-51:
11 |  (libraries hwLib llvm llvm.bitreader llvm.irreader str))
                                           ^^^^^^^^^^^^^
Error: Library "llvm.irreader" not found.
Hint: try:
  dune external-lib-deps --missing src/main.exe
Makefile:8: recipe for target 'all' failed
make: *** [all] Error 1

How can I solve this?

**Comment by Standchen** [comment]
I think your installation wasn't successful. Try:

opam init --compiler=4.11.0 --disable-sandboxing
opam switch create cs448-4.11.0 4.11.0
eval $(opam env)
opam install dune llvm.10.0.0 ounit

**Comment by Junseon00** [comment]
You'd better follow the guide line that we provided.
(https://docs.google.com/presentation/d/1y5Pk-Ks3883ut5WqYRnj8YggxBiAK3ZVzf-hJN6Z8XY/edit?usp=sharing)

**Comment by hsKim25** [comment]
I tried given documents several times in KCLOUD, but I was stuck with install dune. There were several errors.
But anyway, I successfully finished to make HW environment through VM, so I will close this issue.
Thank you all.
